### ISSUE TITLE
txt2vid hangs at random point in generation

### ISSUE THREAD

**Issue by maciekpoplawski** [volotat/SD-CN-Animation#89]
I've tried to run longer generations - 1200 frames. One was successful, three next tries hanged at random point (about 900frame +/- 100). Ram is at 5/24gb, cuda 0%. Is there a way to try to debug it? Or is there a way to save unfinished video? For now i lost like 3x2h of generations :( Maybe some way to store png frames? Also there is no way to interrupt hanged process :(

**Comment by thefixinhixon** [comment]
Seems like the new version has an option to save frames, but I have issues with this as well. Seems to be an issue with vid2vid as well. Nothing in the console window to give any clues to what is going on either unfortunately. I seem to get to around 200-300 frames in before it dies.

**Comment by maciekpoplawski** [comment]
v0.9 Frame: 1149 / 2040; Time elapsed: 2:49:22; Time left: 2:11:20; with checked "Save frames into a folder nearby a video (check it before running the generation if you also want to save frames separately)" .... not any image saved anywhere :( shit this hurts.

**Comment by maciekpoplawski** [comment]
yep... the frames are saved.... after successful generation. To bad i didn't check before putting bigger work :(

**Comment by maciekpoplawski** [comment]
and i just rebooted the pc cause it fixes everything. And ... now the frames are saved when they are generated :D don't want to close this because the generations are still hanging at random point. But for now im super happy about png save option

**Comment by volotat** [comment]
@maciekpoplawski I'm glad that it helps, but I do not know how to help you with hanging issue as without any errors it is really hard to know the source of the problem.

**Comment by maciekpoplawski** [comment]
> @maciekpoplawski I'm glad that it helps, but I do not know how to help you with hanging issue as without any errors it is really hard to know the source of the problem.

i would try to add a tone of prints and maybe i would pinpoint some part of the code.
